diff --git a/node_modules/react-doctor/dist/cli.js b/node_modules/react-doctor/dist/cli.js
index 3820c0a..769e85c 100755
--- a/node_modules/react-doctor/dist/cli.js
+++ b/node_modules/react-doctor/dist/cli.js
@@ -1204,7 +1204,7 @@ const runKnip = async (rootDirectory) => {
 	} else knipResult = await runKnipWithOptions(rootDirectory);
 	const { issues } = knipResult;
 	const diagnostics = [];
-	for (const unusedFile of issues.files) diagnostics.push({
+	for (const unusedFile of (Symbol.iterator in Object(issues.files) ? issues.files : Object.keys(issues.files))) diagnostics.push({
 		filePath: path.relative(rootDirectory, unusedFile),
 		plugin: "knip",
 		rule: "files",
diff --git a/node_modules/react-doctor/dist/index.js b/node_modules/react-doctor/dist/index.js
index 7a05cad..a5177aa 100644
--- a/node_modules/react-doctor/dist/index.js
+++ b/node_modules/react-doctor/dist/index.js
@@ -868,7 +868,7 @@ const runKnip = async (rootDirectory) => {
 	} else knipResult = await runKnipWithOptions(rootDirectory);
 	const { issues } = knipResult;
 	const diagnostics = [];
-	for (const unusedFile of issues.files) diagnostics.push({
+	for (const unusedFile of (Symbol.iterator in Object(issues.files) ? issues.files : Object.keys(issues.files))) diagnostics.push({
 		filePath: path.relative(rootDirectory, unusedFile),
 		plugin: "knip",
 		rule: "files",
